<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableEditDialog</class>
 <widget class="QDialog" name="TableEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="0" colspan="4">
    <widget class="QTextEdit" name="cnAbstractTextEdit">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="20" column="1" colspan="3">
    <widget class="QLineEdit" name="remarkLineEdit"/>
   </item>
   <item row="21" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QTextEdit" name="abstractTextEdit">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="codeLineEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="yearLineEdit"/>
   </item>
   <item row="18" column="0" colspan="4">
    <widget class="QLineEdit" name="conceptLineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ID: </string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Code Link: </string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Chinese Abstract: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Year: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Chinese Title: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="docLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Conference: </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Title: </string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Document Link: </string>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Remarks: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="cnTitleLineEdit"/>
   </item>
   <item row="13" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="4">
    <widget class="QGroupBox" name="conceptCardBox">
     <property name="title">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="titleLineEdit"/>
   </item>
   <item row="12" column="1" colspan="3">
    <widget class="QLineEdit" name="noteLineEdit"/>
   </item>
   <item row="8" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Note Link: </string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Abstract: </string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="confLineEdit"/>
   </item>
   <item row="19" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="2" colspan="2">
    <widget class="QRadioButton" name="conceptRadioManual">
     <property name="text">
      <string>Manual Edit</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QRadioButton" name="conceptRadioCard">
     <property name="text">
      <string>Use Question Cards To Edit</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="4">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Concepts: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="idLineEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TableEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TableEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
